VisualRegressionTracker/VisualRegressionTracker.cs

```diff
@@ -200,6 +203,11 @@ protected async ValueTask Stop(CancellationToken? cancellationToken)
 
             var result = await SubmitTestRun(dto, cancellationToken).ConfigureAwait(false);
 
+            if (result.Status == TestRunStatus.AutoApproved)
+            {
+                return result;
+            }
+
             if (!config.EnableSoftAssert && result.Status != TestRunStatus.Ok)
```

Comment:
could you update it here ?
result.Status != TestRunStatus.Ok or TestRunStatus.AutoApproved

Reply:
Not sure :-)
I have kept it as it, if there is a better way please let me know